nuxeo-services/nuxeo-platform-dublincore/src/main/java/org/nuxeo/ecm/platform/dublincore/constants/DublinCoreConstants.java

```diff
@@ -44,6 +44,11 @@ private DublinCoreConstants() {
 
     public static final String DUBLINCORE_LAST_CONTRIBUTOR_PROPERTY = "dc:lastContributor";
 
+    /**
+     * @since 11.1
+     */
+    public static final String DUBLINCORE_TITLE_PROPERTY = "dc:title";
```

Comment:
Not sure we want to bring nuxeo-platform-dublincore in a hotfix just to add a constant/

Reply:
Agreed

Reply:
Declared it in the test then